lib/katello/tasks/upgrades/4.9/update_custom_products_enablement.rake

```diff
@@ -3,6 +3,7 @@ namespace :katello do
     namespace '4.9' do
       desc "Update custom products enablement"
       task :update_custom_products_enablement => ['environment'] do
+        next unless ::Katello::ProductContent.custom.where(enabled: true).exists?
```

Comment:
I've never seen `next` used outside a loop.. I'd think it'd be equivalent to `return`, right?

Reply:
```
[vagrant@centos8-katello-devel-stable foreman]$ irb
irb(main):001:0> next
Traceback (most recent call last):
        1: from (irb)
SyntaxError ((irb):1: Can't escape from eval with next)
```

Reply:
https://medium.com/tilthat/ruby-block-return-break-next-101815cda709

"`next` inside a block returns from the block. "
That being said I am probably better off wrapping it in if

Reply:
updated

Reply:
yeah that makes me feel better! I'm 100% sure what `if` does :smile: